test-framework/src/main/java/org/elasticsearch/test/CompositeTestCluster.java

```diff
@@ -73,7 +73,7 @@ public synchronized void beforeTest(Random random, double transportClientRatio)
                 try {
                     externalNodes[i] = externalNodes[i].start(client, defaultSettings, NODE_PREFIX + i, cluster.getClusterName(), i);
                 } catch (InterruptedException e) {
-                    Thread.interrupted();
+                    Thread.currentThread().interrupt();
```

Comment:
I think the test should fail. We don't know what else have happened due to this interrupt nor whether the node is ready.